package.json

```diff
@@ -22,7 +22,8 @@
   "scripts": {
     "build": "webpack --config webpack/prod.config.js",
     "dev": "webpack-serve --config webpack/dev.config.js",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "start": "PORT=8080 API_HOST=http://localhost:4000 webpack-serve --config webpack/dev.config.js"
```

Comment:
@developish - This will change the port we use when testing the front-end, so that code changes auto-refresh? You're using 8080 because that's the [proper port for proxies](https://www.quora.com/What-is-port-8080-used-for)?

Reply:
Yes, but I added this as a separate script for testing with my in-between proxy. The normal script `npm run dev` won't change at all.